pkg/process/exec_conf.go

```diff
@@ -30,6 +30,9 @@ import (
 	"storj.io/storj/internal/version"
 )
 
+// DefaultConfFilename is the default filename used for storing a configuration.
+const DefaultConfFilename = "config.yaml"
```

Comment:
nit: we usually use "cfg" for short of configuration. So maybe, name this `DefaultCfgFilename`.